doc/ref_model/chapters/glossary.md

```diff
@@ -73,7 +74,23 @@ To help guide the reader, this glossary provides an introduction to the terminol
 - **Service Assurance (SA)**: collects alarm and monitoring data. Applications within SA or interfacing with SA can then use this data for fault correlation, root cause analysis, service impact analysis, SLA management, security, monitoring and analytic, etc.
 
 <a name="11.1.4"></a>
-### 1.1.4 Other Referenced Terminology
+### 11.1.4 Container Related Terminology
```

Comment:
Since RA-2 refers to "container runtime" do we need a definition for the same?

Something like, "Container runtime reads the configuration files for a container from a directory structure, uses that information to create a container, launches a process inside the container, and performs other lifecycle actions." (adapted from https://github.com/opencontainers/runtime-spec/blob/master/glossary.md).